### ISSUE TITLE
addHeader/noHeader doesn't work

### ISSUE THREAD

**Issue by Janne252** [Perryvw/TypescriptToLua#156]
Type: Bug
Version: 0.5.0

It seems that `addHeader`/`noHeader` doesn't work (which one is it supposed to be? It seems to exist with 2 names: [CommandLineParser.ts](https://github.com/Perryvw/TypescriptToLua/blob/master/src/CommandLineParser.ts))

#### Steps to reproduce the issue

1. `npm install --g --upgrade typescript-to-lua`
1. Create an empty directory and `cd` into it
3. Create `tsconfig.json`
```js
{
    "include": [
        "./**/*.ts"
    ],
    // doesn't seem to affect compiliation, dontRequireLuaLib still works
    "luaLibImport": "none", 
    "luaTarget": "5.1",
    "noHeader": true,
    "addHeader": false,
    "dontRequireLuaLib": true
}
```
4. create an empty TypeScript file (any name) and save it
5. `tstl -p ./tsconfig.json`
6. Notice the header comment in the resulting `.lua` file

**Comment by lolleko** [comment]
Hey! noHeader was added with some refactoring of the runtime lib. It is not released yet thats why it wont work with `npm install --g --upgrade typescript-to-lua` no Header will replace addHeader.

**Comment by Janne252** [comment]
Hey, thanks for the quick response. Keep up the good work!

**Comment by Perryvw** [comment]
Should be solved now 0.7.0 is published.
